src/node_bindings/era_test_node.rs

```diff
@@ -322,7 +322,7 @@ impl EraTestNode {
         cmd.args(self.args);
 
         if let Some(fork) = self.fork {
-            cmd.arg("fork").arg(fork);
+            cmd.arg("fork").arg("--network").arg(fork);
```

Comment:
the new command for specifying the fork network is `fork --network mainnet` instead of the old one: `fork mainnet`.